atlp-rwanda/hackers-ec-be
# feat: send pdf invoice after buyer payment.

### What does this PR do?
When buyer finished to pay products and make order. Buyer should get invoice as pdf on email.

How should this be manually tested?
1. Add products in the cart.
2. Checkout products and choose payment method to use.
3. After payment check your email you will receive invoice email attached with pdf.
